### ISSUE TITLE
Check box to 'I agree with this camp change' in camp history page is not showing in case of 2 direct supporters

### ISSUE THREAD

**Issue by saniatg** [the-canonizer/canonizer.2.0#1312]
Steps:

1. User A and User B  are supporting “camp 1” and “camp 2”
2. Now login with User C and change parent of “camp 2” to “camp 1” and the change is in review
3. User B will check the check box to 'I agree with this camp change' 

Actual Result: User A will not see the check box to agree the change after User B has accepted the check by ticking the check box
Expected Result: User A must also see the check box as both are direct supporters to the camps

**Comment by sonalizibtek** [comment]
Case 1:  User A , User B and User C  are supporting “camp 1” and User A and User B are supporting  camp 2”.  user C change parent of “camp 2” to “camp 1” then

Actual Result:
- when User A agrees to change ,
- In User B side the check box to agree the changes is not showing .
-  when user A check the agree box , and User B is not able to check the agree box. and In the topic view page , the camp hierarchy is not changed, [ It is showing agreement as parent camp of camp2 , instead of camp1 ]
Expected:
-Even B should agree the changes  , then only the changes should reflect 
-Score and hierarchy should get changed only after all direct supporters agrees to the change
When all direct supporters agrees to change, all common supports should get removed from “camp 1” and support should remains same for “camp 2”

**Comment by saniatg** [comment]
@sunilspr7 The main issue is not fixed. 
Also there is another issue the score count issue after 3rd step: User B will check the check box to 'I agree with this camp change'
Issue 1 - the score count is incorrect
![Screenshot 2022-08-30 at 12 38 59 PM](https://user-images.githubusercontent.com/91254708/187372225-a6737e2f-013a-4f31-8c6e-e85d49008ad5.png)
Issue 2- The score of another camp is gone. Shows 0 which is incorrect
![Screenshot 2022-08-30 at 12 39 24 PM](https://user-images.githubusercontent.com/91254708/187372312-bad6616c-cec1-4476-bd5a-0693a4817e3c.png)

**Comment by saniatg** [comment]
@rupalichavan9860 tested and is working fine. Please confirm

**Comment by rupalichavan9860** [comment]
Verified

**Comment by saniatg** [comment]
@rupalichavan9860 Verified on production and is working fine.

**Comment by rupalichavan9860** [comment]
Verified
